tt_metal/impl/program/program.hpp

```diff
@@ -89,11 +89,13 @@ class Program {
     Program(Program &&other) = default;
     Program& operator=(Program &&other) = default;
 
+    void set_global_id(uint64_t id);
     ~Program();
 
     void construct_core_range_set_for_worker_cores();
 
     const uint64_t get_id() const { return this->id; }
+    const uint64_t get_global_id() const { return this->global_id; }
```

Comment:
Can you clarify what global means here? What is the difference between it and id